lib/install/rust.js

```diff
@@ -20,19 +20,17 @@ var tar = require('tar-fs');
 
 // Internal
 var log = require('../log');
+var remote = require('../remote');
 
-var SDK_PATHS = {
+const SDK_PATHS = {
```

Comment:
Yes, I'd say we should migrate in new code, or code that we're in touch with during patches.